nodejs/week2/nodeJS-week2/src/backend/routes/meals.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express');
+const mealRouter = express.Router();
+const fs = require('fs');
+const mealFile = JSON.parse(fs.readFileSync(__dirname + '/../data/meals.json', 'utf-8'));
+
+mealRouter.get('/', (req, res) => {
+	res.json(mealFile);
+});
+
+mealRouter.get('/:id', (req, res) => {
+	const id = parseInt(req.params.id);
+	const mealsdetails = mealFile.filter(meal => meal.id === id);
+	mealsdetails.length > 0 && typeof id === 'number'
```

Comment:
Try give this boolean a name 
const someDescribingName = mealsdetails.length > 0 && typeof id === 'number'

It will make the code a bit easier to read

Reply:
Hi Benjamin,

Thank you for the feedback.
I have made changes according to the feedback, please review